libraries/standard/mqtt/include/mqtt_lightweight.h

```diff
@@ -262,17 +264,68 @@ MQTTStatus_t MQTT_SerializeConnect( const MQTTConnectInfo_t * const pConnectInfo
                                     size_t remainingLength,
                                     const MQTTFixedBuffer_t * const pBuffer );
 
-MQTTStatus_t MQTT_SubscriptionPacketSize( const MQTTSubscribeInfo_t * const pSubscriptionList,
+/**
+ * @brief Get the size and Remaining Length of an MQTT SUBSCRIBE packet.
```

Comment:
(minor) packet size and remaining length.

Reply:
Thanks. Will update